core/quorum/block_voting.go

```diff
@@ -223,14 +224,19 @@ func (bv *BlockVoting) run(strat BlockMakerStrategy) {
 
 				switch e := event.Data.(type) {
 				case downloader.StartEvent: // begin synchronising, stop block creation and/or voting
+					bv.syncChain = true
 					bv.synced = false
 					strat.Pause()
 				case downloader.DoneEvent, downloader.FailedEvent: // caught up, or got an error, start block createion and/or voting
+					bv.syncChain = false
 					bv.synced = true
 					strat.Resume()
 				case core.ChainHeadEvent: // got a new header, reset pending state
-					bv.resetPendingState(e.Block)
+					if !bv.syncChain {
+						bv.synced = true
+					}
 					if bv.synced {
```

Comment:
@bas-vk in a multi-blockmaker configuration, synced will always be false since we aren't passing the singleblockmaker flag so the network will remain on block 0.